lib/if.h

```diff
@@ -301,33 +301,37 @@ RB_HEAD(if_index_head, interface);
 RB_PROTOTYPE(if_index_head, interface, index_entry, if_cmp_func)
 DECLARE_QOBJ_TYPE(interface)
 
-#define IFNAME_RB_INSERT(vrf, ifp)                                             \
-	if (RB_INSERT(if_name_head, &vrf->ifaces_by_name, (ifp)))              \
+#define IFNAME_RB_INSERT(_vrf, _ifp)                                           \
+	if (RB_INSERT(if_name_head, &(_vrf)->ifaces_by_name, (_ifp)))	\
```

Comment:
Let's keep the trailing slash in line with the other slashes

Reply:
ok